tests/python/common/test_subgraph.py

```diff
@@ -95,6 +95,48 @@ def test_subgraph():
     assert F.allclose(h, g.ndata["h"])
 
 
+def test_subgraph_not_relabel_nodes():
+    g = generate_graph()
+    h = g.ndata["h"]
+    l = g.edata["l"]
+    nid = [0, 2, 3, 6, 7, 9]
+    sg = g.subgraph(nid, relabel_nodes=False)
+    eid = {2, 3, 4, 5, 10, 11, 12, 13, 16}
+    assert set(F.asnumpy(sg.edata[dgl.EID])) == eid
+    eid = sg.edata[dgl.EID]
+    # the subgraph is empty initially except for NID/EID field
+    assert len(sg.edata) == 2
+    sh = sg.ndata["h"]
+    assert F.allclose(
+        F.gather_row(h, F.tensor([0, 1, 2, 3, 4, 5, 6, 7, 8, 9])), sh
+    )
+    """
+    s, d, eid
```

Comment:
What's this?

Reply:
The s,d,eid means the source node, destination node and edge id of the subgraph we extract using subgraph() function. The edges labeled 1 are those selected by the subgraph.